<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IMC</class>
 <widget class="QMainWindow" name="IMC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IMC</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resourse.qrc">
    <normaloff>:/imc.png</normaloff>:/imc.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(63, 165, 157);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="styleSheet">
       <string notr="true">font: 75 8pt &quot;Verdana&quot;;
color: rgb(0, 0, 0);
background-color: rgb(95, 158, 160);</string>
      </property>
      <property name="title">
       <string>RESULTADOS OBTENIDOS</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="1" column="0" colspan="2">
        <widget class="QWidget" name="widget_2" native="true">
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>91</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>IMC (kg/mÂ²)</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="outIMC">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>10</y>
            <width>111</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>351</width>
            <height>80</height>
           </rect>
          </property>
          <property name="title">
           <string>Grafica</string>
          </property>
          <widget class="QWidget" name="widget_4" native="true">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>31</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 255, 255);</string>
           </property>
          </widget>
          <widget class="QWidget" name="widget_5" native="true">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>20</y>
             <width>81</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(85, 255, 0);</string>
           </property>
           <widget class="Line" name="outLinea">
            <property name="geometry">
             <rect>
              <x>13</x>
              <y>-20</y>
              <width>16</width>
              <height>61</height>
             </rect>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 0, 0);</string>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="widget_6" native="true">
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>20</y>
             <width>41</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 85, 0);</string>
           </property>
          </widget>
          <widget class="QWidget" name="widget_7" native="true">
           <property name="geometry">
            <rect>
             <x>290</x>
             <y>20</y>
             <width>41</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 0, 0);</string>
           </property>
          </widget>
          <widget class="QWidget" name="widget_8" native="true">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>20</y>
             <width>41</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 127);</string>
           </property>
          </widget>
          <widget class="QWidget" name="widget_9" native="true">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>20</y>
             <width>41</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 85, 127);</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>50</y>
             <width>351</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>15    16        18.5           25        30     35      40</string>
           </property>
          </widget>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>0</y>
            <width>211</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Nivel de peso:</string>
          </property>
         </widget>
         <widget class="QLabel" name="outimc">
          <property name="geometry">
           <rect>
            <x>340</x>
            <y>10</y>
            <width>171</width>
            <height>20</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QWidget" name="widget_3" native="true">
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>55</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Altura:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="outAltura">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>113</width>
            <height>24</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>298</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="2" column="1">
           <widget class="QLineEdit" name="outMin">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="styleSheet">
             <string notr="true">font: 8pt &quot;Verdana&quot;;</string>
            </property>
            <property name="text">
             <string>Actual:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="outMax">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Peso min:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Peso max:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="outActual">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QWidget" name="widget_10" native="true">
         <property name="styleSheet">
          <string notr="true">font: 8pt &quot;Verdana&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>IDENTIFICACION</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Nombre :Angel Pozo</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Cedula: 1753688017</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Grupo: 3 </string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Fecha: 02/02/2022</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">font: 75 8pt &quot;Verdana&quot;;
color: rgb(0, 0, 0);
background-color: rgb(95, 158, 160);</string>
      </property>
      <property name="title">
       <string>INGRESO DE DATOS</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0" rowspan="2" colspan="3">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="3" column="0" colspan="2">
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">font: 8pt &quot;Verdana&quot;;</string>
            </property>
            <property name="text">
             <string>Ingrese su peso actual:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QSpinBox" name="inPeso">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">font: 8pt &quot;Verdana&quot;;</string>
            </property>
            <property name="text">
             <string>Ingrese su estatura (en cm):</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">font: 8pt &quot;Verdana&quot;;</string>
            </property>
            <property name="text">
             <string>Ingrese la fecha:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QSpinBox" name="inAltura">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="maximum">
             <number>210</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDateEdit" name="dateEdit">
            <property name="styleSheet">
             <string notr="true">font: 8pt &quot;Verdana&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>245</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1" colspan="2">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>201</width>
           <height>245</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="btnCalcular">
         <property name="styleSheet">
          <string notr="true">font: 75 8pt &quot;Verdana&quot;;</string>
         </property>
         <property name="text">
          <string>&amp;Calcular</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>98</width>
           <height>52</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1036</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Archivo">
    <property name="title">
     <string>&amp;Archivo</string>
    </property>
    <addaction name="action_Nuevo"/>
    <addaction name="action_Abrir"/>
    <addaction name="action_Guardar"/>
    <addaction name="action_Calcular"/>
    <addaction name="action_Salir"/>
   </widget>
   <addaction name="menu_Archivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Nuevo">
   <property name="text">
    <string>&amp;Nuevo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Abrir">
   <property name="text">
    <string>&amp;Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Guardar">
   <property name="text">
    <string>&amp;Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Calcular">
   <property name="text">
    <string>&amp;Calcular</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_Salir">
   <property name="text">
    <string>&amp;Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resourse.qrc"/>
 </resources>
 <connections/>
</ui>
